code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -28,8 +28,12 @@ Borg Hypospray
 	var/bypass_protection = 0 //If the hypospray can go through armor or thick material
 
 	var/list/datum/reagents/reagent_list = list()
-	var/list/reagent_ids = list(/datum/reagent/medicine/c2/convermol, /datum/reagent/medicine/c2/libital, /datum/reagent/medicine/c2/multiver, /datum/reagent/medicine/c2/aiuri, /datum/reagent/medicine/epinephrine, /datum/reagent/medicine/spaceacillin, /datum/reagent/medicine/salglu_solution)
-	var/accepts_reagent_upgrades = TRUE //If upgrades can increase number of reagents dispensed.
+	var/list/reagent_ids = list(
+		/datum/reagent/medicine/c2/convermol, /datum/reagent/medicine/c2/libital, /datum/reagent/medicine/c2/multiver,
+		/datum/reagent/medicine/c2/aiuri, /datum/reagent/medicine/epinephrine, /datum/reagent/medicine/spaceacillin,
+		/datum/reagent/medicine/salglu_solution
+	)
+	var/list/additional_reagent_ids = null
```

Comment:
Codedocs please.

Reply:
Can you elaborate what you want here.

Reply:
Codedocs, aka `/// Documentation comment explaining the purpose and usage of this variable`.  https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md for details.